src/NexusMods.App.UI/App.axaml

```diff
@@ -13,6 +12,9 @@
     </Application.Styles>
 
     <Application.Resources>
+        <ResourceInclude x:Key="Colors" Source="avares://NexusMods.App.UI/Palatte/Colors.axaml"/>
+        <ResourceInclude x:Key="Fonts" Source="avares://NexusMods.App.UI/Palatte/Fonts.axaml"/>
+
```

Comment:
The idea is that we have everything in a single file that then references the rest. 
We want to be able to swap out one file and get a completely different styling for the app.

Is that possible?

Reply:
You can just load a different resource file later and it will overwrite the previous values.